components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -3183,14 +3184,14 @@ private void parseAllowCrossTenantIntrospection(OMElement oauthConfigElem) {
                 ConfigElements.ALLOW_CROSS_TENANT_TOKEN_INTROSPECTION));
         if (allowCrossTenantIntrospectionElem != null) {
             allowCrossTenantIntrospection = Boolean.parseBoolean(allowCrossTenantIntrospectionElem.getText());
-        } else {
-            /* If config is null, the property must be set to the default value which is true to
-        ensure backward compatibility. */
-            allowCrossTenantIntrospection = true;
         }
     }
 
-    public boolean isCrossTenantTokenInspectionAllowed() {
```

Comment:
Rather than removing this public method, I feel we should deprecate this method. Have we captured these config changes in the migration doc?

Reply:
In here, not removing a method. Only added a block comment above to the method.